fiftyone/server/mutation.py

```diff
@@ -97,7 +97,7 @@ async def set_sidebar_groups(
         sidebar_groups: t.List[SidebarGroupInput],
     ) -> bool:
         state = get_state()
-        view = get_dataset_view(dataset, stages=stages)
+        view = get_view(dataset, stages=stages)
```

Comment:
@kaixi-wang I didn't get a chance to review the saved view PRs where `get_view()` was renamed to `get_dataset_view()`, but I personally find that change to be unnecessary; it's a function that returns a view (saved or unsaved); no need to change it?

I discovered this rename had occurred in some other branches when merging in latest work. In general I think we'll minimize developer pain by not renaming public methods unless there's a really good reason, so mainly using this rename as an excuse to talk about that.

Reply:
I agree it's generally better to avoid renaming public methods, but this method is only defined in and used in the server code, where `view` is often used interchangeably for a dataset view or a serialized view. I think the benefit of the clarity and knowing that you are dealing with the dataset view as opposed to just stages in this scope would actually be  minimizing developer pain.

Reply:
Hmm I see. In my opinion, "view" should always mean `DatasetView`, so the other methods you're referring to (ones that work with "view stages") are the ones whose names are confusing.